docs/book/src/faq/faq.md

```diff
@@ -0,0 +1,43 @@
+# FAQ
+
+Frequently Ask Question
+
+## How to use Kubernetes client in webhook?
+
+You can declare the Kubernetes client in webhook from the controller manager easily.
+
+<details>
+  <summary>Read more</summary>
+
+To begin, you need to import the client and declare the variable:
+
+```go
+import (
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	C client.Client
```

Comment:
I do not think that it should be exposed. You are using only here, right?

Reply:
Yes, you're right!